#!/usr/bin/env python3
PACKAGE = "reu_lane_follow_pkg"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
                        # level,       defaut, min, max
gen.add("hue_l", double_t, 0, "hue low",  25, 0, 179)
gen.add("hue_h", double_t, 0, "hue high", 35, 0, 179)

gen.add("sat_l", double_t, 0, "sat low",  0,   0, 255)
gen.add("sat_h", double_t, 0, "sat high", 255, 0, 255)

gen.add("val_l", double_t, 0, "val low",  0,   0, 255)
gen.add("val_h", double_t, 0, "val high", 255, 0, 255)

exit(gen.generate(PACKAGE, "detect_yellow", "DetectYellow"))
# The 3rd param "FollowLane" must be this filename